Source/CarthageKit/Xcode.swift

```diff
@@ -878,6 +878,23 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 		}
 
 		return schemeSignals
+			// For no shared scheme errors in GitHub repositories, suggest
+			// filing a GitHub issue.
```

Comment:
I think this is reasonably self-documenting to where we might not need this comment, but if we wanted to make it even clearer, maybe the innards of the `case` statement could be pulled out into a function—something like `addIssueFilingSuggestionToError`?